### PR TITLE
Ci graphql server

### PR DESCRIPTION
**Pull request by Valecant**
## Proposed changes

Describe the big picture of your changes here to communicate to the
maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to
that issue.

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation (improve documentation)
- [x] Continuous Integration (functionality that improve automation or testing)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [ ] I have read the [CONTRIBUTING][contrib] doc
- [ ] I have signed the [Contributor License Agreement][cla]
- [ ] I have updated the [RELEASE NOTES][release]
- [ ] I have added tests that prove my fix is effective or that my
      feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion
by explaining why you chose the solution you did and what alternatives
you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/auth-management-ui/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/auth-management-ui/blob/master/CONTRIBUTING.md
    "Contributing to auth-management-ui"
[release]: https://github.com/orchestracities/auth-management-ui/blob/master/RELEASE_NOTES.md
    "auth-management-ui Release Notes"

### CODE REVIEW

**Comment by Valecant** [comment]
To test Graphql is **mandatory** to set up the Configuration API
first, once completed:

```bash
$ cd configuration-api
$ npm install
$ npm test
```
The results of the tests will be visible on the console

**Comment by chicco785** [comment]
include test as part of githubactions

File: configuration-api/main/mongo/populateDB.js [hunk 1]
```diff
@@ -13,41 +13,50 @@ const TenantConfig = new mongoose.Schema({
...
+          }
         }
-      }
-    );
-  }
-});
+      );
+    }
+  });
+});
```
Comment (chicco785) [inline configuration-api/main/mongo/populateDB.js:62]: add return after

File: configuration-api/main/config.js [hunk 2]
```diff
@@ -34,7 +34,16 @@ function loadConfig() {
   } else {
     newConfig.mongo_db = 'mongodb://localhost:27017/graphql';
   }
-
+  if (process.env.REACT_APP_OIDC_CLIENT) {
```
Comment (chicco785) [inline configuration-api/main/config.js:37]: move this code to the test parts

**Review by chicco785 — APPROVED** [review]

**Pull request merged**